### PR TITLE
Link to Quick Start in README

### PR DESCRIPTION
**Pull request by arjun-krishna1**
Add link to Quick Start documentation in readme
https://github.com/8090-inc/xrx-core/issues/14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Build apps with "any modality input (x), reasoning (R), any modality output (x)." 
 
+> **Quick Start.**  We walk through the steps to get the xRx system up and running [here](https://8090-inc.github.io/xrx-core/).
```
Comment (sabbyanandan) [inline README.md:5]: @arjun-krishna1: This is a good change; thank you! However, I would suggest moving it after "## Introduction", and make it a H2 with "## Quick Start" heading.

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
@arjun-krishna1: Thanks. A nit I left as a comment.

File: README.md (hunk 1 above)
Comment (arjun-krishna1) [reply README.md]: Thank you @sabbyanandan , updated!

File: README.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ xRx is a framework for building AI-powered reasoning systems that interact with
 
 We believe that the future of software interactions lies in multimodal experiences, and xRx is at the forefront of this movement. It enables developers to build sophisticated AI systems that seamlessly integrate various input and output modalities, providing users with a truly immersive experience.
 
+## Quick Start
+We walk through the steps to get the xRx system up and running [here](https://8090-inc.github.io/xrx-core/docs/quickstart).
```
Comment (sabbyanandan) [inline README.md:12]:
```suggestion
To get the xRx system up and running, follow our comprehensive [quickstart guide](https://8090-inc.github.io/xrx-core/docs/quickstart).
```

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
Love it. Another nit.

File: README.md (hunk 2 above)
Comment (arjun-krishna1) [reply README.md]: Looks good!

Comment (arjun-krishna1) [reply README.md]: @sabbyanandan looks good!

Comment (arjun-krishna1) [reply README.md]: Updated

**Review by sabbyanandan — APPROVED** [review]
:shipit:

**Comment by arjun-krishna1** [comment]
Thanks @sabbyanandan !

**Pull request merged**